server/lib/middlewares/customMiddles.js

```diff
@@ -0,0 +1,81 @@
+import { NotFoundError } from 'auth0-extension-tools';
+import { getScript } from '../scripts';
+
```

Comment:
Use `lru-memoizer` to cache the scripts for 30 sec. Eg: https://github.com/auth0/auth0-authorization-extension/blob/7a1dc129a287bb6f4eda51e29c5c906c541d716e/server/lib/middlewares/managementClient.js#L21-L32